tensorflow_probability/python/mcmc/transformed_kernel_test.py

```diff
@@ -41,12 +41,35 @@ def _maybe_seed(seed):
   return seed
 
 
+def transform_then_step(bijector):
```

Comment:
let's name this `make_transofrm_then_adapt_kernel`, and the other one below, similarly